src/utils/install/journal_defaults.json

```diff
@@ -3074,6 +3074,21 @@
             "default": ""
           }
     },
+    {
+        "group": {
+            "name": "general"
+        },
+        "setting": {
+            "description": "Enable if you want to hide all review data from authors until released.",
+            "is_translatable": false,
+            "name": "hide_review_data_pre_release",
+            "pretty_name": "Hide Review Data from Authors Before Release",
+            "type": "boolean"
+        },
+        "value": {
+            "default": "on"
+          }
+    },
```

Comment:
Just thinking about how folks might misunderstand this (because they will!), I wonder if we could make the description read something like this: 

Enable if you want to hide all review data from authors through the end of the review stage, except reviews that are individually shared with authors during review.